server/app-engine/static/vehicle-assignments.js

```diff
@@ -624,7 +624,7 @@ function handleCloseButtonPress() {
         }
 
         for (var i=currentAssignments.length-1; i>=0; i--) {
-            if (currentAssignments[i].vehicleID === closeButtonData.item.vehicle) {
+            if (currentAssignments[i].blockID === closeButtonData.item.label) {
                 currentAssignments.splice(i, 1);
                 break;
             }
```

Comment:
Looking at this and the above change... blockID and vehicleID can both be equal to a label value? Doesn't seem like an issue (since both values appear on a button) but just double checking

Reply:
Yes, they'r all strings